packages/video_player/video_player_platform_interface/pubspec.yaml

```diff
@@ -19,3 +19,4 @@ dev_dependencies:
   flutter_test:
     sdk: flutter
   pedantic: ^1.10.0
+  pigeon: ^0.1.21
```

Comment:
This pigeon version should be pegged to a specific version since the wire format isn't guaranteed between versions.